cmd/helm/install.go

```diff
@@ -210,10 +210,15 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 					Getters:          p,
 					RepositoryConfig: settings.RepositoryConfig,
 					RepositoryCache:  settings.RepositoryCache,
+					Debug:            true,
```

Comment:
Should the value of `Debug` be `settings.Debug` instead of `true`?

Reply:
Oh, I guess it should actually be omitted. Good catch

Reply:
I think I will add this as settings.Debug, since that is probably what was originally intended. There was no Debug setting on this before, and I set it to `true` to see if it generated any output (which it doesn't)

Reply:
Fixed. Please re-review when you have a moment.